IntegrationTests/package-manager/test-spm/Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version:5.3
```

Comment:
probably swift 5.0, the same as CombineX?

Reply:
fixed